### ISSUE TITLE
The example uses `uint256` while `Socket` uses `uint32` for `chainSlug()`

### ISSUE THREAD

**Issue by gpersoon** [SocketDotTech/socket-DL#175]
**Describe the bug**
The example uses `uint256` while `Socket` uses `uint32` for `chainSlug()`.
The main `ISocket.sol` also misses the definition for `chainSlug()`.

**To Reproduce**
Check out the following code:
[socketDL-examples ISocket.sol#L79](https://github.com/SocketDotTech/socketDL-examples/blob/main/src/interfaces/ISocket.sol#L79):
```solidity
function chainSlug() external view returns (uint256 chainSlug);
```

[socket-DL SocketBase.sol#L19](https://github.com/SocketDotTech/socket-DL/blob/master/contracts/socket/SocketBase.sol#L19):
```solidity
uint32 public immutable chainSlug;
```

[socket-DL ISocket.sol](https://github.com/SocketDotTech/socket-DL/blob/master/contracts/interfaces/ISocket.sol)
no definition for `chainSlug()`

**Expected behavior**
- the same type should be used everywhere (`uint32`)
- `chainSlug()` should be present to [socket-DL ISocket.sol](https://github.com/SocketDotTech/socket-DL/blob/master/contracts/interfaces/ISocket.sol)

**Additional context**
-

**Comment by arthcp** [comment]
Agree, need to upgrade examples to match the latest socket interfaces.
